docs/MAINTAINING.md

```diff
@@ -44,3 +44,80 @@ do?**
 A: We are always open for suggestions on how to make the process for
 contributors or maintainers smooth. If you have any feedback, please open a
 GitHub ticket for discussion.
+
+## Nominating new maintainers
+
+The nomination process is open to all current maintainers. Nominations are
+discussed in private, using
+[GitHub Team Discussions](https://docs.github.com/en/github/setting-up-and-managing-organizations-and-teams/about-team-discussions).
+(Don't confuse Team Discussions with recently introduced project-level
+[Discussions](https://docs.github.com/en/discussions).)
+
+1. To nominate a new maintainer, post a new comment to the Team Discussion of
+   the team you want the new maintainer to join. If you are not sure which team
```

Comment:
Is there a list of teams for reference?

Reply:
I'll add a link to a query to find all teams a person belongs to.